src/types/api/schema.ts

```diff
@@ -85,6 +85,81 @@ export interface TDirEntry {
     Version?: TPathVersion;
 }
 
+interface TDomainDescription {
+    ProcessingParams?: TProcessingParams;
+
+    DomainKey?: TDomainKey;
+
+    StoragePools: TStoragePool[];
+
+    /** uint64 */
+    PathsInside?: string;
+    /** uint64 */
+    PathsLimit?: string;
+    /** uint64 */
+    ShardsInside?: string;
+    /** uint64 */
+    ShardsLimit?: string;
+
+    ResourcesDomainKey?: TDomainKey;
+
+    DiskSpaceUsage?: TDiskSpaceUsage;
+
+    /** uint64 */
+    PQPartitionsInside?: string;
+    /** uint64 */
+    PQPartitionsLimit?: string;
+
+    DomainState?: TDomainState;
+}
+
+interface TDomainKey {
+    /** fixed64 */
+    SchemeShard?: string;
+    /** fixed64 */
+    PathId?: string;
+}
+
+interface TProcessingParams {
+    Version?: number;
+    /** uint64 */
+    PlanResolution?: string;
+    /** fixed64 */
+    Coordinators: string[];
+    /** uint64 */
+    TimeCastBucketsPerMediator?: string;
+    /** fixed64 */
+    Mediators: string[];
```

Comment:
`repeated` fields in protobuf are also optional

Reply:
I assumed, that in case of empty data we will get empty array, as these fields goes without `optional`. Or it could be a case, when there won't be these fields?

Reply:
in protobuf, an empty array and the absence of a field are indistinguishable
they can be omitted